services/QuillLMS/app/controllers/teachers/classrooms_controller.rb

```diff
@@ -76,8 +80,12 @@ def update
   end
 
   def destroy
-    @classroom.destroy
-    redirect_to teachers_classrooms_path
+    authorize_owner! { params[:id] }
```

Comment:
Since this does use `:id`, should we just add this to the `only` list on line 6 rather than making the call explicit at the top of the action?